CrashLib/CrashLib.xcodeproj/project.pbxproj

```diff
@@ -650,333 +729,69 @@
 /* Begin XCBuildConfiguration section */
 		8065B57D1F4C3FF4004FF622 /* Debug */ = {
 			isa = XCBuildConfiguration;
+			baseConfigurationReference = C2B2102421B6A9FF00F116BD /* macOS.xcconfig */;
 			buildSettings = {
-				CLANG_ANALYZER_NONNULL = YES;
-				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
-				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
-				CODE_SIGN_STYLE = Automatic;
-				COMBINE_HIDPI_IMAGES = YES;
-				DEBUG_INFORMATION_FORMAT = dwarf;
-				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = "";
-				DYLIB_COMPATIBILITY_VERSION = 1;
-				DYLIB_CURRENT_VERSION = 1;
-				DYLIB_INSTALL_NAME_BASE = "@rpath";
-				FRAMEWORK_VERSION = A;
 				INFOPLIST_FILE = CrashLib/Info.plist;
```

Comment:
I was just wondering why `INFOPLIST_FILE` and `PRODUCT_BUNDLE_IDENTIFIER` are kept in the project configuration?

Reply:
The settings like team, provision profile, etc are ignored because it's easier to maintain it from the editor